internal/report/report_test.go

```diff
@@ -51,3 +56,252 @@ func TestAddResult(t *testing.T) {
 		t.Errorf("expected Summary.Error == 1")
 	}
 }
+
+func TestFindClusterPolicyReportResult(t *testing.T) {
+	report := NewClusterPolicyReport("clusterwide")
+	admReviewPass := admv1.AdmissionReview{
+		Request: &admv1.AdmissionRequest{},
+		Response: &admv1.AdmissionResponse{
+			UID:     "4264aa6a-2d4a-49e6-aed8-156d74678dde",
+			Allowed: true,
+		},
+	}
+	policy := policiesv1.ClusterAdmissionPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster-policy",
+		},
+		Spec: policiesv1.ClusterAdmissionPolicySpec{
+			PolicySpec: policiesv1.PolicySpec{
+				BackgroundAudit: true,
+				Rules: []admissionregistrationv1.RuleWithOperations{{
+					Operations: []admissionregistrationv1.OperationType{admissionregistrationv1.Create},
+					Rule: admissionregistrationv1.Rule{
+						APIGroups:   []string{""},
+						APIVersions: []string{"v1"},
+						Resources:   []string{"namespaces"},
+					},
+				},
+				},
+			},
+		},
+	}
+	policy.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   constants.KubewardenPoliciesGroup,
+		Version: constants.KubewardenPoliciesVersion,
+		Kind:    constants.KubewardenKindClusterAdmissionPolicy,
+	})
+
+	var expectedResource unstructured.Unstructured
+	for i := 0; i < 5; i++ {
+		namespaceResource := unstructured.Unstructured{Object: map[string]interface{}{
+			"apiVersion": "v1",
+			"kind":       "Namespace",
+			"metadata": map[string]interface{}{
+				"name":              "testingns" + fmt.Sprint(i),
+				"creationTimestamp": nil,
+			},
+			"spec":   map[string]interface{}{},
+			"status": map[string]interface{}{},
+		},
+		}
+		report.AddResult(&policy, namespaceResource, &admReviewPass, nil)
+		expectedResource = namespaceResource
+	}
+
+	result, ok := report.GetResult(&policy, expectedResource)
+	if !ok {
+		t.Errorf("Policy result not found")
```

Comment:
We should add needed fields to all the logs: policy & resource for result not found, report name for report not found, etc (this applies to the rest of log calls).